pkg/modules/api/api.go

```diff
@@ -478,10 +490,19 @@ func (a *Api) Start() error {
 
 	// As the following code is blocking, run it in a goroutine.
 	go func() {
-		server := &http2.Server{}
-		err := a.srv.StartH2CServer(fmt.Sprintf(":%d", a.port), server)
-		if !errors.Is(err, http.ErrServerClosed) {
-			a.logger.Fatal(err.Error())
+		if a.tlsCertFile != "" && a.tlsKeyFile != "" {
+			// Start an HTTPS server (supports HTTP/2).
+			err := a.srv.StartTLS(fmt.Sprintf(":%d", a.port), a.tlsCertFile, a.tlsKeyFile)
+			if !errors.Is(err, http.ErrServerClosed) {
+				a.logger.Fatal(err.Error())
+			}
+		} else {
+			// Start an HTTP/2 Cleartext (non-HTTPS) server.
+			server := &http2.Server{}
+			err := a.srv.StartH2CServer(fmt.Sprintf(":%d", a.port), server)
+			if !errors.Is(err, http.ErrServerClosed) {
+				a.logger.Fatal(err.Error())
+			}
 		}
```

Comment:
```suggestion
                var error
		if a.tlsCertFile != "" && a.tlsKeyFile != "" {
			// Start an HTTPS server (supports HTTP/2).
			err = a.srv.StartTLS(fmt.Sprintf(":%d", a.port), a.tlsCertFile, a.tlsKeyFile)
		} else {
			// Start an HTTP/2 Cleartext (non-HTTPS) server.
			server := &http2.Server{}
			err = a.srv.StartH2CServer(fmt.Sprintf(":%d", a.port), server)
		}
		if !errors.Is(err, http.ErrServerClosed) {
			a.logger.Fatal(err.Error())
		}
```

Reply:
I think this should be `var err` instead of `var error`, right?